src/app/controllers/users.py

```diff
@@ -33,6 +40,65 @@ def list_user_per_page(users):
 
     return jsonify(list_users_dict), 200
 
+@user.route("/create", methods = ['POST'])
+@requires_access_level(['READ', 'WRITE', 'UPDATE', 'DELETE'])
+def post_create_users():
+    
+    list_keys = ['gender', 'city', 'role', 'name', 'age', 'email',\
+        'phone', 'password', 'cep', 'district', \
+        'street', 'number_street']
+
+    data = allkeys_in(request.get_json(), list_keys)
+
+    if "error" in data:
+        return jsonify(data), 400
+
+    get_gender = Gender.query.filter(Gender.description.ilike(f"%{data['gender']}%")).first()
+    get_city = City.query.filter(City.name.ilike(f"%{data['city']}")).all()
+    get_role = Role.query.filter(Role.name.ilike(f"%{data['role']}%")).first()
+
+    if type(get_gender) == NoneType:
+        return jsonify({"error": "Genêro não existe no banco de dados."}), 404
+
```

Comment:
Testei o colocando outras palavras ou até vazio em "gender" no body mas a execução passa direto pelo if e vai a função create na variável response, que retorna o erro

Reply:
Resolvido os imports de share_schema. E essa questão de validação no body fica pendente enquanto a PR do Thiago referente não for mergeada. Irei mexer quando estiver ok